scrypto/src/macros.rs

```diff
@@ -262,7 +262,7 @@ macro_rules! extern_blueprint_internal {
             $($method_contents:tt)*
         }
     ) => {
-        #[derive(Copy, Clone, Debug, Eq, PartialEq)]
+        #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
```

Comment:
Haha the `send_batch` method is one example where the claimants is an `IndexMap<Global<Account>, ResourceSpecifier>` and so the `Global<Account>` needs to implement `Hash` ([`IndexMap` impl](https://docs.rs/indexmap/latest/src/indexmap/map.rs.html#378-399))

Reply:
Hmm but I would expect this to be the `Global` struct which implements Hash (which is the wrapper around the address), rather than the Account itself (which is a handle).

Perhaps `Global` needs to implement Hash manually so it doesn't fall foul of the "default require all generic params to also implement X" behaviour of the Rust inbuilt derive macros?

Reply:
Sounds good, I've implemented `Hash` on `Global` manually in my new PR. Implementation is simple: the hash of `Global<T>` is the hash of the node id of that global object.